docs/addon-host-access.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+title: "Host-acces addon"
+permalink: /docs/addon-host-access
+---
+
+# Host-access addon
+
+The host-access addon enables access of services running on the host machine via a fixed IP.
+This becomes useful when your machine changes IPs as you hope through different networks. 
+
+You can install this addon with:
+
+```bash
+microk8s enable host-access
+```
+
+The local network interface created is named `lo:microk8s` and the IP assigned by default is `10.0.1.1`.
```

Comment:
A new local network interface named `lo:microk8s` is created with a default IP address of `10.0.1.1`.